tests/benchmarks/i2c_endless/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_NRFX_TWIS22=y
```

Comment:
Support for PDK can be remove for this tests - no need to keep pdk files, only dk can stay.

Reply:
done